### PR TITLE
Creating initial additional resources page

### PR DESCRIPTION
**Pull request by miohtama**
### What I did

I created a new documentation page that contains links to the useful Vyper resources outside the official Vyper repos.

### How I did it

I mostly used the content from Vyper section that I have maintained at ethereum.org now and then.

### How to verify it

See that Sphinx docs get the new page built:

image

### Commit message

Creating initial additional resources page

### Description for the changelog

Creating initial additional resources page

### Cute Animal Picture

```
                                           *##X        
                                           *###*       
                                           ####X       
             *#                           ####X#       
             X#X                          #X##X#       
 X           *##                          ###X#X       
X#     *XX*  *##X                         X####        
##*  *######XX# #                         *##X         
### *#X     #X* #              **          ##          
#X###X      X  ##          ***####         ##*         
#X  XX      *###X        #####X  X#       *#X*         
 ##X#*       *  ##*     *#   X    X#      XXX*         
X#XX#* X*X  *##XX#X     X#         X#     #*#          
#####* #*#*  #X*X*      #X          #X   X# #          
 * *#*       ##*      *##       X#* *##*## XX          
   X#XXX***  #*########*      X####  ##** X#           
 *##*     ***#X X*   *X*      *##### *#XX##*           
 XX           #*       *X      #####**#XX*    ##       
 #X **        *#    *X          X### ##   *X*####      
 #X*##* X#X    #X X####          X#  *#X  ###X  ###    
 #X*### ###X   XX X####X  *XXXX       *#XX#      *#X   
 X# ##X X##X   XX  X####X #####         ##X *##X  #X   
 X#  *   X#    #X X#####X #####             #XX#* *#   
  ##*         *#  ######X #####*           *#  X####*  
   ###XX***XX##X  ######X *####*           *#    ***   
    *X######X**    #####*  ####            ##          
         #*         ####   *X*            ##           
         #*         *XX*         X*     *#X            
         #*                     X*X#XXXX#X             
         X#                    #*  **###               
         X#* *               *#     * X#               
         #X#XX*            *##     X##X#               
         #  X##       X   ####X*  ##**X                
        *#   X#*     X# X###X*##X ##                   
        #X  X##X    X####X #XX##X#**                   
       #X  X# *#   *#X     *##* ##                     
       #  *#* *#   #X                                  
      *#   #  #X  ##                                   
      X# X##  #*  X#                                   
     *# *#*   #*  X#                                   
   X##X #*    #X X#*                                   
   #X   #     #XX#*                                    
  X# *  XX   ## #*                                     
  ##X   #*  #X  #*                                     
  *##*X##  X#*  #X       The Dutch Dude                
   X###X   #X*  #*                                     
           #X  X#                                      
           X###X*                                      
             **
```

### CODE REVIEW

File: docs/resources.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+examples, courses and other learning material.
+
+General
+-------
+
+-  `More Vyper by Example <https://vyper-by-example.org/> by Smart Contract Engineer`__
+-  `Vyper cheat Sheet <https://reference.auditless.com/cheatsheet>`__
+-  `Vyper Hub for development <https://github.com/zcor/vyper-dev>`__
+-  `Vyper greatest hits smart contract examples <https://github.com/pynchmeister/vyper-greatest-hits/tree/main/contracts>`__
+-  `Awesome Vyper curated resources <https://github.com/spadebuilders/awesome-vyper>`__
```
Comment (fubuloubu) [inline docs/resources.rst:16]: This one is unmaintained

File: docs/resources.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Frameworks and tooling
+----------------------
+
+- `ApeWorX - The Ethereum development framework for Python Developers, Data Scientists, and Security Professionals <https://www.apeworx.io/>`__
+- `Forge development framework <https://www.alchemy.com/dapps/foundry-forge>`__
```
Comment (fubuloubu) [inline docs/resources.rst:22]: Forge doesn't have support for Vyper out of the box, probably better linking that hack that let's it work instead

File: docs/resources.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Other resources and learning material
+#####################################
+
+Vyper has an active community. You can find third party tutorials,
+examples, courses and other learning material.
+
+General
+-------
+
```
Comment (fubuloubu) [inline docs/resources.rst:11]:
```suggestion

-  `Ape Academy <https://academy.apeworx.io/> by ApeWorX`__
```

Comment (fubuloubu) [reply docs/resources.rst:11]: Because we have a lot of Vyper content

File: docs/resources.rst (hunk 1 above)
Comment (miohtama) [reply docs/resources.rst:16]: I will create a "Unmaintained" section for old resources. They might still become handy for some.

File: docs/resources.rst (hunk 2 above)
Comment (miohtama) [reply docs/resources.rst:22]: Alchemy advertised Vyper support for me, so maybe I got fooled here 🤨

**Comment by miohtama** [comment]
@fubuloubu Updated the PR based on the discussion

File: docs/resources.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- `ApeWorX - The Ethereum development framework for Python Developers, Data Scientists, and Security Professionals <https://www.apeworx.io/>`__
```
Comment (fubuloubu) [inline docs/resources.rst:12]:
```suggestion
-  `Ape Academy <https://academy.apeworx.io/> by ApeWorX`__
```

Comment (fubuloubu) [reply docs/resources.rst:12]: Since you mentioned Ape Framework below

**Review by fubuloubu — APPROVED** [review]

**Comment by miohtama** [comment]
Bump on this one. I think someone needs to merge this.

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**